vsintegration/Vsix/VisualFSharpTemplates/VisualFSharpTemplates.csproj

```diff
@@ -7,7 +7,7 @@
     <TargetFramework>net46</TargetFramework>
     <OutputType>Library</OutputType>
     <ExtensionInstallationFolder>Microsoft\FSharpTemplates</ExtensionInstallationFolder>
-    <DeployExtension>false</DeployExtension>
+    <DeployExtension>true</DeployExtension>
```

Comment:
This line can be removed; it was simply overriding [this](https://github.com/Microsoft/visualfsharp/blob/66af44bcf9fd94e2f88263cd1723d0b3bb995056/vsintegration/Vsix/Directory.Build.props#L8-L9).

Reply:
Sweet.